routes/crop/crop-routes.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Why has this entire file been touched?  What happened here?  Where is the `getCrops` function?